Source/Tools/FEXLoader/LinuxSyscalls/SignalDelegator.cpp

```diff
@@ -2020,7 +2030,7 @@ namespace FEX::HLE {
     return Result == -1 ? -errno : Result;
   }
 
-  fextl::unique_ptr<FEX::HLE::SignalDelegator> CreateSignalDelegator(FEXCore::Context::Context *CTX) {
-    return fextl::make_unique<FEX::HLE::SignalDelegator>(CTX);
+  fextl::unique_ptr<FEX::HLE::SignalDelegator> CreateSignalDelegator(FEXCore::Context::Context *CTX, fextl::string const *ApplicationName) {
```

Comment:
`Program.ProgramPath` doesn't change after SignalDelegator is created, is it? We should just pass the string in by copy instead then, to avoid this quasi-global and the lifetime dependencies it implicitly builds.

Reply:
Converted over to a copy.